fe/fe-core/src/main/java/org/apache/doris/catalog/Column.java

```diff
@@ -186,6 +188,11 @@ public void createChildrenColumn(Type type, Column column) {
             Column c = new Column(COLUMN_ARRAY_CHILDREN, ((ArrayType) type).getItemType());
             c.setIsAllowNull(((ArrayType) type).getContainsNull());
             column.addChildrenColumn(c);
+        } else if (type.isMapType()) {
+            Column k = new Column(COLUMN_MAP_KEY, ((MapType) type).getKeyType());
+            Column v = new Column(COLUMN_MAP_VALUE, ((MapType) type).getValueType());
```

Comment:
Here no need to setIsAllowNull for the column k and v?

Reply:
column K, V in columnMap should not be null ?